packages/apps/composer-app/src/playwright/comments.spec.ts

```diff
@@ -94,7 +103,11 @@ test.describe('Comments tests', () => {
   });
 
   test('delete thread', async () => {
+    await host.closeToast();
```

Comment:
why does this start with close toast?

Reply:
The observability toast was getting in the way. Wait, shouldn't our host setup be closing that?

Reply:
I have it setup so that it shouldn't even be fired when running e2e tests https://github.com/dxos/dxos/blob/jdw%2Fgraph-opt/packages/apps/plugins/plugin-observability/src/ObservabilityPlugin.tsx#L87

Reply:
Broken link. I was seeing it when debugging and inspecting tests.

Reply:
sorry, updated link to point to main